INSTALL.md

```diff
@@ -78,17 +96,93 @@ locale: en
 #running in a load-balanced environment, these need to be the same on all the webservers in
 #the group. IMPORTANT:  You MUST change this to your own value as using the default value would
 #be EXTREMELY dangerous in a production enviroment!
-token: ThisCanBeWhateverYouLike,JustMakeSureYouChangeIt!
+secret: ThisCanBeWhateverYouLike,JustMakeSureYouChangeIt!
 
 #These should all be set to 'false' in a production environment
 debug_toolbar: false
 debug_redirects: false
-user_assetic_controller: false
+
+# can currently be 'form' (default),'ldap', or 'shibboleth' depeding on your institution
+authentication_type: form
+
+#legacy_password_salt should be your old Ilios2 password salt, if upgrading from ilios 2, otherwise, leave as 'null' (default):
+legacy_password_salt: null
+
+#file_system_storage_path is where you plan to put your uploadable files, like learning materials
+file_system_storage_path: /web/ilios3/file_uploads
+
+
+# if you are using LDAP to authenticate users on the system, you should fill these in, otherwise leave 'null'
+ldap_authentication_host: null
+ldap_authentication_port: null
+ldap_authentication_bind_template: null
+ldap_directory_url: null
+ldap_directory_user: null
+ldap_directory_password: null
+ldap_directory_search_base: null
+ldap_directory_campus_id_property: null
+ldap_directory_username_property: null
+
+# This is for the curriculum inventory reporting
+# This is the name of the institution as it will appear in the Curriculum Inventory Export
+institution_domain: ucsf.edu
+# This should be the link to where your Curriculum Inventory Export may be viewed:
+supporting_link: null
+
+#and finally, the current mysql database version
+database_mysql_version: 5.1
 ```   
-6. Update the config.yml file in the Ilios application config folder (/web/ilios3/ilios/app/config/config.yml) to reflect the latest revision of the Ilios frontend.  For more info on this process, please see Ilios 3 Frontend Deployment  
-7. Now change into the document root folder ('/web/ilios3/ilios/web') and remove the 'app_dev.php' file as this file should not exist on a production machine.
+
+## Database
+
+### Upgrading from previous versions of Ilios
+
+If you are already running a 2.x version of Ilios, you can upgrade your current database by following the intructions at [Upgrading Ilios 2.x to Ilios 3](https://github.com/ilios/ilios/blob/master/UPGRADE.md).
+
+### Creating a new database for Ilios
+If you are NOT upgrading from a previous version of Ilios, you can create a new, empty database schema by using the following Symfony console command:
+
+```bash
+bin/console doctrine:database:create
+bin/console doctrine:migrations:migrate
+bin/console doctrine:fixtures:load
+```
+
+This will create all structure data for the database, including tables and constraints, but there will not be any data until you login and add some!
```

Comment:
Actually, this should say something more like " this create your database schema, with all tables and constraints, and will also load in all the default lookup table data, like competencies and topics --- which you can modify once you're done with setup --- but it won't have any course data or any other unique data about your specific school or curriculum until you log in and add some."